src/olympia/devhub/views.py

```diff
@@ -1147,6 +1147,12 @@ def version_edit(request, addon_id, addon, version_id):
                 if (isinstance(form, forms.CompatForm) and
                         'max' in form.changed_data):
                     _log_max_version_change(addon, version, form.instance)
+
+        has_changed = [f in version_form.changed_data for f in (
+            'source', 'approvalnotes')]
```

Comment:
Style wise I'd do something like

```python
fields = ('source', 'approvalnotes')
has_changed = [field in version_form.changed_data for field in fields]
```

I think it's a bit easier to read but that's just a nitpick.